api/STYLE.md

```diff
@@ -113,3 +113,66 @@ for services and configs should be `//docs` (proto documentation tool).
 
 Extensions should use the regular hierarchy. For example, configuration for network filters belongs
 in a package under `envoy.config.filter.network`.
+
+## Adding an extension configuration to the API
+
+Extensions must currently be added as v2 APIs following the package organization above.
+To add an extension config to the API, the steps below should be followed:
+
+1. Place the v2 extension configuration `.proto` in `api/config`, e.g.
+   `api/config/filter/http/foobar/v2/foobar.proto` together with an initial BUILD file:
+   ```
+   load("@envoy_api//bazel:api_build_system.bzl", "api_proto_package")
+
+   licenses(["notice"])  # Apache 2
+
+   api_proto_package(
+       deps = ["@com_github_cncf_udpa//udpa/annotations:pkg"],
+       )
+   ```
+1. Add to the v2 extension config proto `import "udpa/annotations/migrate.proto";`
+2. Add to the v2 extension config proto a package level `option (udpa.annotations.file_migrate).move_to_package = "envoy.extensions.filters.http.foobar.v3alpha";`.
```

Comment:
nit: can you reference somehow the new layout that is described in the previous section?